hinmil/test/ParserSpec.hs

```diff
@@ -0,0 +1,16 @@
+--{-# LANGUAGE OverloadedStrings #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module ParserSpec (spec) where
+
+--import Control.Applicative hiding (some)
+import Test.Hspec
+import Test.Hspec.Megaparsec
```

Comment:
Wow, I didn't even notice this and `shouldParse` during my first review. This rings a bell from long ago. Good work.